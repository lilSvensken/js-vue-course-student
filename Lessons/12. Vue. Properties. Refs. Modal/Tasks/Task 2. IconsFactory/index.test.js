/**
 * @jest-environment jsdom
 */

const { mount } = require("@vue/test-utils");
const ItisIcon = require("./ItisIcon.vue");

describe("12.c2 ItisIcon", () => {
    const wrapper = mount(ItisIcon.default);
    expect(wrapper.element).toMatchSnapshot();

    it("12.c2.2 ItisIcon is a Vue instance", () => {
        expect(wrapper.vm).toBeTruthy();
    });

    it("12.c2.3 change prop 'icon-type' to 'excited'", async () => {
        await wrapper.setProps({ iconType: 'excited' });
        expect(wrapper.element).toMatchSnapshot();
    });

    it("12.c2.4 change prop 'icon-type' to 'loved'", async () => {
        await wrapper.setProps({ iconType: 'loved' });
        expect(wrapper.element).toMatchSnapshot();
    });

    it("12.c2.5 change prop 'icon-type' to 'happy'", async () => {
        await wrapper.setProps({ iconType: 'happy' });
        expect(wrapper.element).toMatchSnapshot();
    });
});
