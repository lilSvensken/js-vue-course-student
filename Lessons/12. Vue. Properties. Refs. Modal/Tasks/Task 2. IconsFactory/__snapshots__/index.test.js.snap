// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[` 1`] = `
<svg
  height="94.572px"
  viewBox="0 -12.02 94.572 94.572"
  width="94.572px"
  xmlns="http://www.w3.org/2000/svg"
>
  <g
    id="surprised"
    transform="translate(-348.019 -53.014)"
  >
    <path
      d="M349.442,87.785c.283-26.649,16.426-33.362,45.857-33.353,29.458.009,45.585,6.732,45.869,33.353.293,27.433-16.714,34.458-46.565,34.333C365.116,121.992,349.158,114.549,349.442,87.785Z"
      data-name="Path 53"
      fill="#fdb991"
      fill-rule="evenodd"
      id="Path_53"
    />
     
    <path
      d="M349.442,87.785c.015-1.428.078-2.8.184-4.11,1.853,22.4,17.57,28.84,44.977,28.957,27.8.116,44.46-5.971,46.38-28.97.106,1.319.17,2.69.186,4.123.292,27.433-16.715,34.458-46.566,34.333C365.116,121.992,349.158,114.55,349.442,87.785Z"
      data-name="Path 54"
      fill="#1a1818"
      fill-rule="evenodd"
      id="Path_54"
      opacity="0.15"
    />
     
    <path
      d="M348.023,87.774c.147-13.863,4.477-22.577,12.649-27.858,8.008-5.175,19.647-6.907,34.627-6.9s26.629,1.745,34.642,6.925c8.172,5.282,12.5,13.991,12.646,27.835.152,14.26-4.252,23.255-12.624,28.7-8.211,5.341-20.175,7.124-35.366,7.06-15.02-.064-26.638-2.02-34.54-7.422-8.051-5.5-12.181-14.431-12.034-28.34Zm14.185-25.466C354.88,67.043,351,75.01,350.86,87.8c-.136,12.855,3.571,21.031,10.8,25.971,7.377,5.043,18.483,6.871,32.949,6.932,14.66.062,26.125-1.606,33.808-6.6,7.52-4.893,11.474-13.128,11.334-26.3-.136-12.776-4.017-20.74-11.344-25.477-7.484-4.838-18.638-6.464-33.107-6.469C380.849,55.846,369.7,57.467,362.208,62.308Z"
      data-name="Path 55"
      fill="#1a1818"
      fill-rule="evenodd"
      id="Path_55"
    />
     
    <path
      d="M371.561,82.775a6.226,6.226,0,1,0-6.226-6.226A6.238,6.238,0,0,0,371.561,82.775Z"
      data-name="Path 56"
      fill="#1a1818"
      fill-rule="evenodd"
      id="Path_56"
    />
     
    <path
      d="M370.966,76.158a2,2,0,1,0-2-1.995A2,2,0,0,0,370.966,76.158Z"
      data-name="Path 57"
      fill="#fff"
      fill-rule="evenodd"
      id="Path_57"
    />
     
    <path
      d="M368.868,79.478a1.24,1.24,0,1,0-1.241-1.241A1.243,1.243,0,0,0,368.868,79.478Z"
      data-name="Path 58"
      fill="#fff"
      fill-rule="evenodd"
      id="Path_58"
    />
     
    <g
      data-name="Group 14"
      id="Group_14"
    >
      <path
        d="M419.049,82.775a6.226,6.226,0,1,0-6.226-6.226A6.238,6.238,0,0,0,419.049,82.775Z"
        data-name="Path 59"
        fill="#1a1818"
        fill-rule="evenodd"
        id="Path_59"
      />
       
      <path
        d="M418.455,76.158a2,2,0,1,0-2-1.995A2,2,0,0,0,418.455,76.158Z"
        data-name="Path 60"
        fill="#fff"
        fill-rule="evenodd"
        id="Path_60"
      />
       
      <path
        d="M416.356,79.478a1.24,1.24,0,1,0-1.24-1.241A1.243,1.243,0,0,0,416.356,79.478Z"
        data-name="Path 61"
        fill="#fff"
        fill-rule="evenodd"
        id="Path_61"
      />
    </g>
     
    <g
      data-name="Group 15"
      id="Group_15"
    >
      <path
        d="M374.588,88.89l-6.057,0a1.68,1.68,0,0,1-1.171-2.884,5.753,5.753,0,0,1,7.927-.459,5.538,5.538,0,0,1,.568.568,1.678,1.678,0,0,1-1.267,2.773Z"
        data-name="Path 62"
        fill="#eb505e"
        fill-rule="evenodd"
        id="Path_62"
      />
       
      <path
        d="M422.078,88.892H416.02A1.68,1.68,0,0,1,414.857,86a5.581,5.581,0,0,1,.462-.449,5.77,5.77,0,0,1,7.458,0,5.474,5.474,0,0,1,.567.56,1.68,1.68,0,0,1-1.266,2.782Z"
        data-name="Path 63"
        fill="#eb505e"
        fill-rule="evenodd"
        id="Path_63"
      />
    </g>
     
    <path
      d="M395.3,97.191c4.19,0,7.608-3.979,7.608-8.857s-3.418-8.857-7.608-8.857-7.608,3.978-7.608,8.857S391.115,97.191,395.3,97.191Z"
      data-name="Path 64"
      fill="#1a1818"
      fill-rule="evenodd"
      id="Path_64"
    />
     
    <path
      d="M367.106,68.335a1.419,1.419,0,0,1-2.146-1.857,8.407,8.407,0,0,1,.869-.871,8.879,8.879,0,0,1,11.458,0,8.455,8.455,0,0,1,.873.872,1.419,1.419,0,0,1-2.146,1.857,5.542,5.542,0,0,0-.584-.583,6.036,6.036,0,0,0-7.744,0A5.5,5.5,0,0,0,367.106,68.335Z"
      data-name="Path 65"
      fill="#1a1818"
      fill-rule="evenodd"
      id="Path_65"
    />
     
    <path
      d="M414.594,68.335a1.419,1.419,0,0,1-2.146-1.857,8.321,8.321,0,0,1,.87-.871,8.879,8.879,0,0,1,11.458,0,8.552,8.552,0,0,1,.873.872,1.419,1.419,0,0,1-2.146,1.857,5.737,5.737,0,0,0-.584-.583,6.036,6.036,0,0,0-7.744,0A5.508,5.508,0,0,0,414.594,68.335Z"
      data-name="Path 66"
      fill="#1a1818"
      fill-rule="evenodd"
      id="Path_66"
    />
  </g>
</svg>
`;

exports[`12.c2 ItisIcon 12.c2.3 change prop 'icon-type' to 'excited' 1`] = `
<svg
  height="94.56px"
  viewBox="0 -12.02 94.56 94.56"
  width="94.56px"
  xmlns="http://www.w3.org/2000/svg"
>
  <g
    id="excited"
    transform="translate(-490.634 -446.908)"
  >
    <path
      d="M492.056,481.676c.283-26.646,16.425-33.358,45.852-33.349,29.455.008,45.58,6.731,45.864,33.349.293,27.43-16.713,34.454-46.56,34.329C507.728,515.879,491.772,508.437,492.056,481.676Z"
      data-name="Path 112"
      fill="#9b88bf"
      fill-rule="evenodd"
      id="Path_112"
    />
     
    <path
      d="M492.056,481.675c.016-1.427.079-2.794.184-4.109,1.853,22.4,17.568,28.837,44.972,28.954,27.8.116,44.455-5.971,46.375-28.967.106,1.32.17,2.69.185,4.122.293,27.431-16.713,34.454-46.56,34.33C507.728,515.879,491.772,508.438,492.056,481.675Z"
      data-name="Path 113"
      fill="#1a1818"
      fill-rule="evenodd"
      id="Path_113"
      opacity="0.15"
    />
     
    <path
      d="M490.638,481.665c.147-13.861,4.476-22.575,12.647-27.855,8.007-5.174,19.645-6.906,34.623-6.9s26.626,1.745,34.639,6.925c8.17,5.281,12.495,13.989,12.643,27.832.152,14.259-4.251,23.253-12.623,28.7-8.208,5.34-20.172,7.122-35.361,7.059-15.018-.064-26.635-2.021-34.536-7.421-8.05-5.5-12.179-14.429-12.032-28.337ZM504.821,456.2c-7.327,4.734-11.211,12.7-11.347,25.485-.136,12.853,3.571,21.028,10.8,25.968,7.376,5.042,18.481,6.87,32.945,6.932,14.658.061,26.122-1.606,33.8-6.6,7.52-4.891,11.473-13.125,11.333-26.3-.136-12.774-4.016-20.738-11.343-25.474-7.484-4.838-18.636-6.464-33.1-6.468C523.46,449.741,512.312,451.361,504.821,456.2Z"
      data-name="Path 114"
      fill="#1a1818"
      fill-rule="evenodd"
      id="Path_114"
    />
     
    <path
      d="M517.2,482.784l-6.057,0a1.679,1.679,0,0,1-1.171-2.883,5.833,5.833,0,0,1,.471-.46,5.769,5.769,0,0,1,7.455,0,5.538,5.538,0,0,1,.568.568,1.678,1.678,0,0,1-1.266,2.773Z"
      data-name="Path 115"
      fill="#eb505e"
      fill-rule="evenodd"
      id="Path_115"
    />
     
    <path
      d="M564.683,482.785h-6.057a1.679,1.679,0,0,1-1.162-2.892,5.439,5.439,0,0,1,.462-.45,5.768,5.768,0,0,1,7.457,0,5.576,5.576,0,0,1,.567.56,1.678,1.678,0,0,1-1.267,2.781Z"
      data-name="Path 116"
      fill="#eb505e"
      fill-rule="evenodd"
      id="Path_116"
    />
     
    <path
      d="M547.813,479.336l0,.217a12.315,12.315,0,0,1-2.862,8.03,9.012,9.012,0,0,1-14.079,0,12.323,12.323,0,0,1-2.865-8.03l.006-.279a.709.709,0,0,1,.709-.7l18.378,0a.713.713,0,0,1,.713.712Z"
      data-name="Path 117"
      fill="#1a1818"
      fill-rule="evenodd"
      id="Path_117"
    />
     
    <path
      d="M514.169,457.235l2.972,6.155,6.772.924-4.935,4.728,1.213,6.726-6.022-3.233-6.022,3.233,1.213-6.726-4.935-4.728,6.772-.924Z"
      data-name="Path 118"
      fill="#ffcd37"
      fill-rule="evenodd"
      id="Path_118"
    />
     
    <g
      data-name="Group 30"
      id="Group_30"
    >
      <path
        d="M514.468,463.941a1.995,1.995,0,1,1-1.995,1.995A1.995,1.995,0,0,1,514.468,463.941Z"
        data-name="Path 119"
        fill="#fff"
        fill-rule="evenodd"
        id="Path_119"
      />
       
      <path
        d="M512.37,471.251a1.241,1.241,0,1,0-1.24-1.241A1.243,1.243,0,0,0,512.37,471.251Z"
        data-name="Path 120"
        fill="#fff"
        fill-rule="evenodd"
        id="Path_120"
      />
    </g>
     
    <g
      data-name="Group 32"
      id="Group_32"
    >
      <path
        d="M561.655,457.235l2.972,6.155,6.772.924-4.935,4.728,1.213,6.726-6.022-3.233-6.022,3.233,1.213-6.726-4.935-4.728,6.773-.924Z"
        data-name="Path 121"
        fill="#ffcd37"
        fill-rule="evenodd"
        id="Path_121"
      />
       
      <g
        data-name="Group 31"
        id="Group_31"
      >
        <path
          d="M561.954,467.931a1.995,1.995,0,1,0-2-1.995A2,2,0,0,0,561.954,467.931Z"
          data-name="Path 122"
          fill="#fff"
          fill-rule="evenodd"
          id="Path_122"
        />
         
        <path
          d="M559.856,471.251a1.241,1.241,0,1,0-1.24-1.241A1.243,1.243,0,0,0,559.856,471.251Z"
          data-name="Path 123"
          fill="#fff"
          fill-rule="evenodd"
          id="Path_123"
        />
      </g>
    </g>
  </g>
</svg>
`;

exports[`12.c2 ItisIcon 12.c2.4 change prop 'icon-type' to 'loved' 1`] = `
<svg
  height="94.571px"
  viewBox="0 -12.02 94.571 94.571"
  width="94.571px"
  xmlns="http://www.w3.org/2000/svg"
>
  <g
    id="loved"
    transform="translate(-490.629 -53.014)"
  >
    <path
      d="M492.051,87.785c.283-26.649,16.427-33.362,45.857-33.353,29.458.009,45.585,6.732,45.87,33.353.292,27.433-16.715,34.458-46.566,34.333C507.725,121.992,491.767,114.549,492.051,87.785Z"
      data-name="Path 68"
      fill="#ecc3dc"
      fill-rule="evenodd"
      id="Path_68"
    />
     
    <path
      d="M492.051,87.785c.016-1.428.079-2.8.184-4.11,1.853,22.4,17.57,28.84,44.977,28.957,27.8.116,44.46-5.971,46.38-28.97.107,1.319.17,2.69.186,4.123.292,27.433-16.715,34.458-46.566,34.333C507.725,121.992,491.767,114.55,492.051,87.785Z"
      data-name="Path 69"
      fill="#1a1818"
      fill-rule="evenodd"
      id="Path_69"
      opacity="0.15"
    />
     
    <path
      d="M490.633,87.774c.147-13.863,4.476-22.577,12.649-27.858,8.008-5.175,19.647-6.907,34.626-6.9s26.629,1.745,34.643,6.925c8.171,5.282,12.5,13.991,12.645,27.835.152,14.26-4.252,23.255-12.624,28.7-8.21,5.341-20.175,7.124-35.366,7.06-15.02-.064-26.638-2.02-34.54-7.422-8.051-5.5-12.181-14.431-12.033-28.34Zm14.184-25.466c-7.328,4.735-11.212,12.7-11.347,25.488-.137,12.855,3.57,21.031,10.8,25.971,7.378,5.043,18.484,6.871,32.949,6.932,14.66.062,26.126-1.606,33.808-6.6,7.521-4.893,11.475-13.128,11.334-26.3-.136-12.776-4.016-20.74-11.343-25.477-7.486-4.838-18.639-6.464-33.108-6.469C523.458,55.846,512.309,57.467,504.817,62.308Z"
      data-name="Path 70"
      fill="#1a1818"
      fill-rule="evenodd"
      id="Path_70"
    />
     
    <path
      d="M517.2,88.893l-6.057,0a1.68,1.68,0,0,1-1.171-2.884,5.534,5.534,0,0,1,.472-.46,5.765,5.765,0,0,1,7.455,0,5.435,5.435,0,0,1,.568.568,1.678,1.678,0,0,1-1.267,2.773Z"
      data-name="Path 71"
      fill="#eb505e"
      fill-rule="evenodd"
      id="Path_71"
    />
     
    <path
      d="M564.687,88.895h-6.058A1.68,1.68,0,0,1,557.466,86a5.6,5.6,0,0,1,.462-.45,5.77,5.77,0,0,1,7.458,0,5.588,5.588,0,0,1,.568.56,1.68,1.68,0,0,1-1.267,2.782Z"
      data-name="Path 72"
      fill="#eb505e"
      fill-rule="evenodd"
      id="Path_72"
    />
     
    <path
      d="M514.167,69.321c1.013-1.866,2.028-2.8,4.058-2.8a3.905,3.905,0,0,1,4.057,3.732c0,3.733-4.057,7.466-8.115,11.2-4.059-3.732-8.116-7.465-8.116-11.2a3.905,3.905,0,0,1,4.057-3.732C512.138,66.523,513.153,67.455,514.167,69.321Z"
      data-name="Path 73"
      fill="#eb505e"
      fill-rule="evenodd"
      id="Path_73"
      stroke="#eb505e"
      stroke-linecap="round"
      stroke-linejoin="round"
      stroke-width="1.549"
    />
     
    <path
      d="M561.657,69.321c1.014-1.866,2.029-2.8,4.059-2.8a3.905,3.905,0,0,1,4.057,3.732c0,3.733-4.057,7.466-8.116,11.2-4.058-3.732-8.115-7.465-8.115-11.2a3.905,3.905,0,0,1,4.057-3.732C559.629,66.523,560.644,67.455,561.657,69.321Z"
      data-name="Path 74"
      fill="#eb505e"
      fill-rule="evenodd"
      id="Path_74"
      stroke="#eb505e"
      stroke-linecap="round"
      stroke-linejoin="round"
      stroke-width="1.549"
    />
     
    <g
      data-name="Group 18"
      id="Group_18"
    >
      <path
        d="M518.428,72.2a2,2,0,1,0-2-1.995A2,2,0,0,0,518.428,72.2Z"
        data-name="Path 75"
        fill="#fff"
        fill-rule="evenodd"
        id="Path_75"
      />
       
      <path
        d="M516.33,75.518a1.24,1.24,0,1,0-1.241-1.241A1.243,1.243,0,0,0,516.33,75.518Z"
        data-name="Path 76"
        fill="#fff"
        fill-rule="evenodd"
        id="Path_76"
      />
    </g>
     
    <g
      data-name="Group 19"
      id="Group_19"
    >
      <path
        d="M565.952,72.2a2,2,0,1,0-2-1.995A2,2,0,0,0,565.952,72.2Z"
        data-name="Path 77"
        fill="#fff"
        fill-rule="evenodd"
        id="Path_77"
      />
       
      <path
        d="M563.853,75.518a1.24,1.24,0,1,0-1.24-1.241A1.243,1.243,0,0,0,563.853,75.518Z"
        data-name="Path 78"
        fill="#fff"
        fill-rule="evenodd"
        id="Path_78"
      />
    </g>
     
    <path
      d="M547.815,85.445l0,.217a12.308,12.308,0,0,1-2.862,8.031,9.014,9.014,0,0,1-14.081,0,12.325,12.325,0,0,1-2.865-8.031l.006-.279a.709.709,0,0,1,.709-.7l18.38,0a.713.713,0,0,1,.713.712Z"
      data-name="Path 79"
      fill="#1a1818"
      fill-rule="evenodd"
      id="Path_79"
    />
  </g>
</svg>
`;

exports[`12.c2 ItisIcon 12.c2.5 change prop 'icon-type' to 'happy' 1`] = `
<svg
  height="94.572px"
  viewBox="0 -12.02 94.572 94.572"
  width="94.572px"
  xmlns="http://www.w3.org/2000/svg"
>
  <g
    id="happy"
    transform="translate(-62.8 -53.014)"
  >
    <path
      d="M64.223,87.785c.283-26.649,16.426-33.362,45.857-33.353,29.458.009,45.585,6.732,45.869,33.353.293,27.433-16.715,34.458-46.566,34.333C79.9,121.992,63.938,114.549,64.223,87.785Z"
      data-name="Path 1"
      fill="#a4d4b2"
      fill-rule="evenodd"
      id="Path_1"
    />
     
    <path
      d="M64.223,87.785c.015-1.428.078-2.8.184-4.11,1.853,22.4,17.569,28.84,44.977,28.957,27.8.116,44.46-5.971,46.379-28.97.107,1.319.171,2.69.186,4.123.292,27.433-16.714,34.458-46.565,34.333C79.9,121.992,63.939,114.55,64.223,87.785Z"
      data-name="Path 2"
      fill="#1a1818"
      fill-rule="evenodd"
      id="Path_2"
      opacity="0.15"
    />
     
    <path
      d="M62.8,87.774c.147-13.863,4.477-22.577,12.649-27.858,8.008-5.175,19.647-6.907,34.627-6.9s26.629,1.745,34.642,6.925c8.172,5.282,12.5,13.991,12.646,27.835.152,14.26-4.252,23.255-12.625,28.7-8.21,5.341-20.175,7.124-35.365,7.06-15.021-.064-26.638-2.02-34.54-7.422-8.051-5.5-12.182-14.431-12.034-28.34ZM76.988,62.308C69.661,67.043,65.776,75.01,65.641,87.8c-.137,12.855,3.57,21.031,10.8,25.971,7.377,5.043,18.483,6.871,32.949,6.932,14.66.062,26.125-1.606,33.808-6.6,7.52-4.893,11.474-13.128,11.334-26.3-.137-12.776-4.017-20.74-11.344-25.477-7.485-4.838-18.638-6.464-33.107-6.469C95.63,55.846,84.48,57.467,76.988,62.308Z"
      data-name="Path 3"
      fill="#1a1818"
      fill-rule="evenodd"
      id="Path_3"
    />
     
    <path
      d="M95.871,86.318a1.419,1.419,0,1,1,2.146-1.857,15.345,15.345,0,0,0,1.58,1.584,16.308,16.308,0,0,0,20.969,0,15.318,15.318,0,0,0,1.588-1.584,1.419,1.419,0,1,1,2.146,1.857,17.937,17.937,0,0,1-1.877,1.873,19.151,19.151,0,0,1-24.683,0A17.964,17.964,0,0,1,95.871,86.318Z"
      data-name="Path 4"
      fill="#1a1818"
      fill-rule="evenodd"
      id="Path_4"
    />
     
    <path
      d="M89.369,88.893l-6.057,0a1.68,1.68,0,0,1-1.172-2.884,5.753,5.753,0,0,1,7.928-.46,5.537,5.537,0,0,1,.568.568,1.678,1.678,0,0,1-1.267,2.773Z"
      data-name="Path 5"
      fill="#eb505e"
      fill-rule="evenodd"
      id="Path_5"
    />
     
    <path
      d="M136.858,88.895H130.8A1.68,1.68,0,0,1,129.638,86a5.438,5.438,0,0,1,.462-.45,5.77,5.77,0,0,1,7.458,0,5.575,5.575,0,0,1,.567.56,1.68,1.68,0,0,1-1.267,2.782Z"
      data-name="Path 6"
      fill="#eb505e"
      fill-rule="evenodd"
      id="Path_6"
    />
     
    <g
      data-name="Group 1"
      id="Group_1"
    >
      <path
        d="M86.342,82.325A6.226,6.226,0,1,0,80.116,76.1,6.237,6.237,0,0,0,86.342,82.325Z"
        data-name="Path 7"
        fill="#1a1818"
        fill-rule="evenodd"
        id="Path_7"
      />
       
      <path
        d="M85.747,75.709a2,2,0,1,0-2-2A2,2,0,0,0,85.747,75.709Z"
        data-name="Path 8"
        fill="#fff"
        fill-rule="evenodd"
        id="Path_8"
      />
       
      <path
        d="M83.649,79.028a1.241,1.241,0,1,0-1.241-1.24A1.242,1.242,0,0,0,83.649,79.028Z"
        data-name="Path 9"
        fill="#fff"
        fill-rule="evenodd"
        id="Path_9"
      />
    </g>
     
    <g
      data-name="Group 2"
      id="Group_2"
    >
      <path
        d="M133.83,82.325A6.226,6.226,0,1,0,127.6,76.1,6.238,6.238,0,0,0,133.83,82.325Z"
        data-name="Path 10"
        fill="#1a1818"
        fill-rule="evenodd"
        id="Path_10"
      />
       
      <path
        d="M133.236,75.709a2,2,0,1,0-2-2A2,2,0,0,0,133.236,75.709Z"
        data-name="Path 11"
        fill="#fff"
        fill-rule="evenodd"
        id="Path_11"
      />
       
      <path
        d="M131.137,79.028a1.241,1.241,0,1,0-1.241-1.24A1.243,1.243,0,0,0,131.137,79.028Z"
        data-name="Path 12"
        fill="#fff"
        fill-rule="evenodd"
        id="Path_12"
      />
    </g>
  </g>
</svg>
`;
